src/test/java/racingcar/fixture/CarsFixture.java

```diff
@@ -0,0 +1,13 @@
+package racingcar.fixture;
+
+import java.util.List;
+import racingcar.domain.Cars;
+
+public class CarsFixture {
+
+    public final static List<String> CAR_NAMES = List.of("car1", "car2", "car3");
```

Comment:
지난번 코멘트에서 `CarFixture.K3` 으로 예시를 드려서 오해할 수 있는 상황이었네요 😢  죄송합니다
Car의 fixture가 아닌 이름에 대한 fixture였습니다
이름도 fixture로 만들어 둔다면 index를 활용해 접근하지 않아도 될 것 같아요 
```suggestion
    public static final List<String> CAR_NAMES = List.of(CarNameFixture.K3, CarNameFixture.K5, CarNameFixture.K7);
```